src/test/java/org/testng/internal/issue2426/SuiteTest.java

```diff
@@ -0,0 +1,25 @@
+package org.testng.internal.issue2426;
+
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import org.testng.xml.SuiteXmlParser;
+import org.testng.xml.XmlSuite;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import test.SimpleBaseTest;
+
+public class SuiteTest extends SimpleBaseTest {
```

Comment:
A reference to this test class should have been added to `src/test/resources/testng.xml` so that the test gets executed for every build.